packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -82,14 +93,24 @@ export const convertRemoteParticipantToVideoGalleryRemoteParticipant = (
     }
   }
 
+  /**
+   * Since VideoTile only cares about the following states,
+   * we don't observe the other states like 'EarlyMedia'.
+   */
+  let participantState: VideoGalleryRemoteParticipantState | undefined;
+  if (state === 'Connecting' || state === 'Ringing' || state === 'Hold') {
```

Comment:
Do we filter out participants with other states anywhere? Or do we show a participant with state "Disconnected" or "Idle" in the video gallery currently?

Reply:
Hmm we just filter out `inLobby`:
![image](https://user-images.githubusercontent.com/2684369/182648301-c1561741-5a6f-4d70-a23c-983a6d32aeb0.png)

(not part of this PR) Should we be filtering out every state we don't explicitly handle?

Reply:
I think you are right.
Pinged the calling sdk team to understand what the states like `Idle` and `EarlyMedia` mean.
Will modify the filter logic once I have answers.

Reply:
Updated the code, we didn't need to filter out states but convert them to video tile states.